app/controllers/pages_controller.rb

```diff
@@ -9,7 +8,7 @@ def show
   end
 
   def index
-    @presenter = PagesPresenter.new(pages, verb_results, adjective_results, search_page, search_params, search_options, from_date_as_datetime, to_date_as_datetime)
+    @presenter = PagesPresenter.new(pages, verb_results, adjective_results, page, search_options, from_date_as_datetime, to_date_as_datetime)
```

Comment:
It may not have had the best name to begin with, but `page` has the potential here to be mistaken for something other than an `int`. Happy to go with what you think is best here.

Reply:
Investigated this and preferred the method in `GenericPhrasesPresenter` of pulling the page from search_options so removed it